src/pages/Homepage/Trendingitem/Trendingitem.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
+import { useWishlist } from "../../../context/wishlistContext";
 import "./Trendingitem.css";
 import Pasta from "../../../assets/pasta.jpg";
```

Comment:
If possible try importing images one statement using index.js file. So that we can reduce the total no of lines in a particular component. try to change the next pr request.